ocaml/forkexecd/lib/forkhelpers.ml

```diff
@@ -325,10 +332,12 @@ let execute_command_get_output_inner ?env ?stdin ?(syslog_stdout = NoSyslogging)
   finally
     (fun () ->
       match
+        with_tracing ~tracing ~name:"with_logfile_out_fd" @@ fun tracing ->
         with_logfile_fd "execute_command_get_out" (fun out_fd ->
+            with_tracing ~tracing ~name:"with_logfile_err_fd" @@ fun tracing ->
```

Comment:
here are we saying that the trace for `logfile_out_fd` is the parent of the trace `logfile_err_fd`? Is this intended? I was thinking about sharing the same parent for them?

Reply:
Yes, it is intended. They are both wrappers, it is just that they are chained. 

This is how it shows visually (the trace is from when they had the same name so ignore that :) ) :
![image](https://github.com/xapi-project/xen-api/assets/48069069/b0de72a9-7377-4325-9036-be87b4dc0b6a)

If they were represented with the same parent, I'd interpret it as 2 separate executed in parallel and not just only one. 

Initially, I wanted to have a `with_tracing` call inside `with_logfile_fd` but I found it harder to read what was happening.

Reply:
or could we just have one `with_tracing` with the name `with_logifle_fd` for these two things? Is there benefit in separating them?